pkgs/applications/editors/emacs/elisp-packages/elpa-devel-packages.nix

```diff
@@ -64,6 +64,25 @@ self: let
         '';
       });
 
+      org = super.org.overrideAttrs (old: {
+        dontUnpack = false;
+        patches = old.patches or [ ] ++ [
+          # security fix backported from 9.7.5
+          (pkgs.fetchpatch {
+            url = "https://git.savannah.gnu.org/cgit/emacs/org-mode.git/patch/?id=f4cc61636947b5c2f0afc67174dd369fe3277aa8";
+            hash = "sha256-bGgsnTSn6SMu1J8P2BfJjrKx2845FCsUB2okcIrEjDg=";
+            stripLen = 1;
+          })
+        ];
+        postPatch = old.postPatch or "" + "\n" + ''
+          cd ..
```

Comment:
Doesn't this break the build?

Reply:
why do you think so?  You can try it out.

Reply:
Don't we change the build dir for all other phases here? How is that supposed to work?

Reply:
> How is that supposed to work?

It actually works, which is verified by the CI.

> Don't we change the build dir for all other phases here?

For `elpaBuild`, working directory has no effect.  It just need a right `src` environment variable pointing to a "good" tarball.

Reply:
It'd still feel cleaner to me to pusd/popd here, even if the build technically works without. Overriders could get confused for instance.

Reply:
We can use `pushd ..` and `popd` here.  However, in that case, deleting the unpacked source directory is not allowed to avoid the failure of `popd`.  In addition, we have to disable the `buildPhase`.

I just think the `pushd` and `popd` is not worth the effect.  Just `cd ..` is good enough.

Reply:
> Overriders could get confused for instance.

Please note that by default `dontUnpack` is true, which means there is no unpacked directory.  So `cd ..` does not change anything actually.

Reply:
Pushed the `pushd`/`popd` version, which has its own side effects.

Reply:
I see, I was missing this context. Feel free to revert to the previous version.

Reply:
fwiw for future reference: you can use a subshell to sidestep all of this:

```bash
(
  cd some/dir
)
```

This way you don't have to juggle directory stacks and you don't have to worry about anything which potentially exits early. (technically it also survives the subshell deleting the working directory of the parent shell tho at that point I'm not sure it's still a good idea 😂)